src/pybamm/models/full_battery_models/lithium_ion/electrode_soh.py

```diff
@@ -855,6 +936,7 @@ def get_min_max_ocps(self, inputs=None):
         return [sol["Un(x_0)"], sol["Un(x_100)"], sol["Up(y_100)"], sol["Up(y_0)"]]
 
     def theoretical_energy_integral(self, inputs, points=1000):
+        direction = "discharge"
```

Comment:
maybe worth a comment somewhere that this is calculated using the discharge branch?